tests/test_e2e_40_sdntrace.py

```diff
@@ -776,7 +776,23 @@ def test_060_run_sdntrace_order(cls):
     def test_070_run_sdntrace_untagged_vlan(cls):
         """Run SDNTrace to test /traces endpoint when vlan is untagged in evc"""
 
-        cls.create_evc("untagged", interface_a="00:00:00:00:00:00:00:02:1", interface_z="00:00:00:00:00:00:00:03:1")
+        payload = {
+            "name": "Vlan_untagged",
+            "enabled": True,
+            "dynamic_backup_path": True,
+            "uni_a": {
+                "interface_id": "00:00:00:00:00:00:00:02:1",
+                "tag": {"tag_type": 1, "value": "untagged"}
+            },
+            "uni_z": {
+                "interface_id": "00:00:00:00:00:00:00:03:1",
+                "tag": {"tag_type": 1, "value": "untagged"}
+            }
+        }
+        api_url = KYTOS_API + '/kytos/mef_eline/v2/evc/'
+        response = requests.post(api_url, json=payload)
+        assert response.status_code in [201, 409], response.text
```

Comment:
@gretelliz asserting for 409 is problematic and will become difficult to reason and maintain, also if you it resulted in 409 you can't be sure what sort of conflict that was, so you never know if the other EVC had the untagged behavior like that you needed here. Ideally, each test case outcome shouldn't rely or interfere with creation from other test cases, currently this test suite is only executing `setup_class` and `teardown_class` so there's margin to still have resources (EVCs) from prior test cases. 

With that said, I'll ask you to identify why this is resulting in 409, and then identify which EVC is that, and consequently try to delete it upfront when it exists. Test execution order should work both when this test is executing individually or with the entire test suite. Can you improve this part accordingly? Thanks

Reply:
@viniarck, I catch you.

As you commented, only `setup_class` and `teardown_class` should be executed before this e2e test_70.

An evc with vlan=400 is created in `setup_class`, so when trying to create an evc with `vlan=untagged`, the error related to the evc already exists should not appear. right?

Anyway, my solution will be to list all evc's to make sure that a new evc is been created.

Wdyt?